source/utilities.tex

```diff
@@ -19886,6 +19886,30 @@
 For floating-point types this field specifies the formatting precision.
 For string types it specifies how many characters will be used from the string.
 
+\pnum
+When the \tcode{L} option is used, the form used for the conversion
+is called the \defn{locale-specific} form.
+The \tcode{L} option is only valid for arithmetic types, and
+its effect depends upon the type.
+\begin{itemize}
+
+\item
+For integral types, the locale-specific form
+causes the context’s locale to be used
+to insert the appropriate digit group separator characters.
+
+\item
+For floating-point types, the locale-specific form
+causes the context’s locale to be used
+to insert the appropriate digit group and decimal radix separator characters.
+
+\item
+For the textual representation of \tcode{bool}, the locale-specific form
+causes the context’s locale to be used
```

Comment:
```suggestion
causes the context's locale to be used
```

Reply:
I guess this a Unicode issue?  can't see any difference on a mac, so thanks for catching these!